src/components/dialogs/ImportAPIDialog.svelte

```diff
@@ -173,10 +173,26 @@
     {/if}
   </form>
 
-  <button slot="footer" class="uk-button uk-button-primary" type="submit" form={id} disabled={loading}>
-    Fetch Data
-    {#if loading}
-      <div uk-spinner />
-    {/if}
-  </button>
+  <div slot="footer">
+    <div class="uk-form-controls uk-form-controls-text container">
+      <button class="uk-button uk-button-primary" type="submit" form={id} disabled={loading}>
+        Fetch Data
+        {#if loading}
+          <div uk-spinner />
+        {/if}
+      </button>
+      <label
+        ><input class="uk-checkbox" type="checkbox" bind:checked={$storeApiKeys} />
+        Save API keys & auth tokens</label
```

Comment:
since its really the same thing:
```suggestion
        Save API key (auth token)</label
```

Reply:
Fixed!

Reply:
thank you, but i apologize -- can we add one more bit for clarity?

```suggestion
        Save API key (auth token) between visits</label
```